### DISCUSSION TITLE
Keep getting this error when trying to play music.

### DISCUSSION THREAD

**Discussion by ethanocurtis** [codetheweb/muse#1011] [Q&A]
ope: 
Invalid prisma.keyValueCache.upsert() invocation:

Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(SqliteFailure(Error { code: ReadOnly, extended_code: 8 }, Some("attempt to write a readonly database"))), transient: false })

Any fix? Thanks!

**Comment by codetheweb** [comment]
Please open an issue with the template and fill in the details, without knowing your configuration I can't really help.
